views/api/index.pug

```diff
@@ -102,13 +102,13 @@ block content
             img(src='https://i.imgur.com/JNNRQSm.png', height=40, style='padding: 0px 10px 0px 0px')
             | Pinterest
     .col-md-4
-      a(href='/api/google-maps', style='color: #fff')
-        .card(style='background-color: #20a360').text-white.mb-3
+      a(href='/api/google-maps', style='color: #000')
+        .card(style='background-color: #20b067').mb-3
```

Comment:
What is the change?  More specifically why #20b067?

Reply:
To add more context on my question.  That page is supposed to use the brand colors.  So I think per https://partnermarketinghub.withgoogle.com/brands/google-maps/visual-identity/color-palette1/ you would need to use #0F9D58.  Let me know if you found 20b067 in some other google branding doc.

Reply:
The reason behind the slight change in the background colour to 20b067 was to obtain a contrast-ratio higher than 7 to conform to WCAG AAA ( [Check the differences between the A, AA, AAA levels](https://ialabs.ie/what-is-the-difference-between-wcag-a-aa-and-aaa/) ). Using #20b067 gave a contrast ratio of 7.46:1, meeting the needs without creating much of a visual difference.

Reply:
I was unaware that the background colours were based off their actual brand colors. If that is to be prioritised over a higher contrast ratio, I am ready to change the background colour to #0F9D58 to match the branch color of Google Maps

Reply:
yes, let's use the google branding